### ISSUE TITLE
No formatting happens with custom .clang-format style

### ISSUE THREAD

**Issue by anktkr** [mapbox/XcodeClangFormat#44]
**Version 1.2**
**Steps:**
- Make a new format-test folder.
- Download the clang-format file from https://github.com/blender/blender/blob/master/.clang-format .
- Make a new Xcode command line tool project. Language C++, place it in the above folder. 
- Move the clang-format file in the same folder as the `main.cpp`. & select this style in XcodeClangFormat app. 
<details><summary>Expand: Badly Formatted Code</summary>

```c++
//
//  main.cpp
//  clang-format-test
//
//  Copyright © 2020 ak. All rights reserved.
//

#include <iostream>

int
main(int argc, const char* argv[])
{
  // Long comment that must be broken down in multiple lines.
  // insert code here... insert code here... insert code here... insert code here... insert code here... insert code here...
  std::cout << "Hello, World!\n";
										  return 0;
}
```
</details>

- Try to format entire file, it won't respond at all.
- Change  the style to one of the standards, it'll format.
----------- 
Although v 1.1 crashed sometimes, it used to work with custom styles.

**Comment by kkaefer** [comment]
Thanks for including so much information in this bug report. I can reproduce the lack of formatting with Blender's clang-format file and it seems like this is because this extension assumes that the fallback template is the "empty" style, while the command-line version of `clang-format` assumes that it's `LLVM`. A fix for that is in https://github.com/mapbox/XcodeClangFormat/pull/45.

I could not reproduce the "badly formatted code" issue though. Are you sure that you're using the most recent version of XcodeClangFormat, which is 1.2 at the time of this writing? macOS' extension system heavily caches the locations of .appex files. If you have the `.app` bundle on your system in multiple locations, Xcode doesn't always pick the one installed in `/Applications`, so make sure that you delete any other versions you may have in other locations.

**Comment by anktkr** [comment]
Thank you for promptly fixing it! The extension saves me quite a lot of app juggling.

> I could not reproduce the "badly formatted code" issue though.

Sorry for leaving it ambiguous. I meant that copy this code in the C++ file, and it should get formatted, which wasn't happening. Is it getting formatted now, i.e. is the comment beyond 99 chars in a line getting broken down into two lines ? 

Also, could you kindly provide an app bundle ? I plan to build llvm myself (for other several tools too), but till then, I cannot build the app myself.

**Comment by kkaefer** [comment]
Ah, got it! I tried it and it does get formatted correctly, with the comment getting broken down over two lines.

I will publish a 1.2.1 release later today.

> I plan to build llvm myself (for other several tools too), but till then, I cannot build the app myself.
XcodeClangFormat's build process downloads the pre-built binaries from llvm.org and doesn't require building Clang itself.

**Comment by anktkr** [comment]
Awesome!

**Comment by kkaefer** [comment]
Published 1.2.1: https://github.com/mapbox/XcodeClangFormat/releases/tag/v1.2.1
